spec/controllers/settings_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe SettingsController, type: :controller do
+  render_views
+  let(:user) { FactoryGirl.create(:user) }
+
+  describe '#update_profile' do
+    let(:username) { 'miyukki' }
+    let(:password) { 'p@ssw0rd' }
+    let(:student_id) { 71349640 }
+    let(:ldap_bind_result) { true }
+    let(:ldap_info_result) {
+      double(:hash, {
+        uid: [username],
+        gecos: ["Yusei Yamanaka, SFC, #{student_id}, t13964yy@sfc.keio.ac.jp"],
+        uidnumber: ['11032'],
+        gidnumber: ['1000'],
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.